Sparkle/SUBinaryDeltaUnarchiver.m

```diff
@@ -6,16 +6,25 @@
 //  Copyright 2009 Mark Rowe. All rights reserved.
 //
 
-#import "SUBinaryDeltaCommon.h"
 #import "SUBinaryDeltaUnarchiver.h"
+#import "SUUnarchiverNotifier.h"
+#import "SUBinaryDeltaCommon.h"
 #import "SUBinaryDeltaApply.h"
-#import "SUUnarchiver_Private.h"
-#import "SUFileManager.h"
-#import "SUHost.h"
 #import "SULog.h"
+#import "SUFileManager.h"
+
+@interface SUBinaryDeltaUnarchiver ()
+
+@property (nonatomic, copy, readonly) NSString *archivePath;
+@property (nonatomic, copy, readonly) NSString *updateHostBundlePath;
+
+@end
 
 @implementation SUBinaryDeltaUnarchiver
 
+@synthesize archivePath = _archivePath;
+@synthesize updateHostBundlePath = _updateHostBundlePath;
```

Comment:
I'd love to without breaking anything ;). The first step would to make sure all code is @synthesize a = _a first before moving to autosynthesizing which would mean some ivars need name changing.